src/types/index.ts

```diff
@@ -895,3 +899,25 @@ export interface IAddressBookEntry {
   isBookmarked: boolean;
   protocol: Protocol;
 }
+
+export interface IOtherSettings {
+  isSeedBackedUp?: boolean;
+  saveErrorLog?: boolean;
+  isBiometricLoginEnabled?: boolean;
+  secureLoginTimeout: number;
+}
+
+export interface IKey {
```

Comment:
I feel that we might have some naming conflict in the future as the crypto related stuff also uses "keys". We had `IKey` in the past that described data required to decrypt blockchain data. I suggest to use more specific name, eg.: `ISecurityKey`.